conda-recipes/llvmdev_manylinux2014/build.sh

```diff
@@ -93,3 +94,45 @@ make check-llvm-unit || exit $?
 
 # From: https://github.com/conda-forge/llvmdev-feedstock/pull/53
 make install || exit $?
+
+# run the tests, skip some on linux-32
+cd ../test
+if [[ $ARCH == 'i686' ]]; then
+    ../build/bin/llvm-lit -vv Transforms Analysis CodeGen/X86
+else
+    ../build/bin/llvm-lit -vv Transforms ExecutionEngine Analysis CodeGen/X86
+fi
```

Comment:
These lines were removed in 981a262023c6aa6c3fb4f1ae22adb6dd7f59822c but I think these re-additions have come from somwhere in #909. I think they can be removed again:

```suggestion
```